geomstats/visualization/hyperbolic.py

```diff
@@ -1,44 +1,33 @@
 """Visualization for Geometric Statistics."""
+import geomstats._backend as gs
 import matplotlib.pyplot as plt
-from mpl_toolkits.mplot3d import Axes3D  # NOQA
-
-import geomstats.backend as gs
 from geomstats.geometry.hyperboloid import Hyperboloid
 from geomstats.geometry.poincare_half_space import PoincareHalfSpace
+from geomstats.visualization._plotting import Plotter
+from mpl_toolkits.mplot3d import Axes3D  # NOQA
 
-H2 = Hyperboloid(dim=2)
-POINCARE_HALF_PLANE = PoincareHalfSpace(dim=2)
-
-
-AX_SCALE = 1.2
 
+class KleinDisk(Plotter):
+    """Class used to plot KleinDisk"""
 
-class KleinDisk:
     def __init__(self, points=None):
+        super().__init__()
+
         self.center = gs.array([0.0, 0.0])
         self.points = []
         if points is not None:
             self.add_points(points)
```

Comment:
one of the main ideas of `Plotter` is to not have state, i.e. `self.points` is to be removed. Therefore, this should be removed.